radarlabs/react-native-radar
# Add expand units field

QA:
methodology: use waypoint's call of auto-complete and manually set expandUnits field to be true and false and observe result
<img width="490" alt="image" src="https://github.com/radarlabs/react-native-radar/assets/139801512/8b630818-7ef7-4977-b600-fe3588f40d6b">

IOS, when field is set to true, we can see that the SDK includes it in the http request
<img width="1425" alt="image" src="https://github.com/radarlabs/react-native-radar/assets/139801512/2b96e1d3-b59a-4c31-98b6-57d372374d3a">

Android, when the field is set to true and the layers is not set, we get a bad request error as expected, setting the field to false subsequently results in a normal call
<img width="822" alt="image" src="https://github.com/radarlabs/react-native-radar/assets/139801512/6385eb99-7bf9-4a91-89bb-abbd2ca47762">
